app/code/local/Potoky/ImageAutoImport/Block/Adminhtml/FormContainer.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Created by PhpStorm.
+ * User: Alex
+ * Date: 4/17/2018
+ * Time: 1:03 PM
+ */
+class Potoky_ImageAutoImport_Block_Adminhtml_FormContainer extends Mage_Adminhtml_Block_Widget_Form_Container
```

Comment:
Potoky_ImageAutoImport_Block_Adminhtml_FormContainer should be Potoky_ImageAutoImport_Block_Adminhtml_Form_Container or Potoky_ImageAutoImport_Block_Adminhtml_Formcontainer

There is no option to use CamelCase names between underscore symbols. There're only 2 exceptions: Module Name and Controllers, e.g ProductController, but not SimpleProductController (should be SimpleproductController, or should be placed in subfolder Simple and it's name be ProductController) .